docs/migration-guides/5.0.md

```diff
@@ -7,7 +7,7 @@ this migration guide covers the most notable changes.
 
 1. `oas2` and `oas3` rulesets have been merged to `oas`
 
-From now on, you don't need to worry about `oas2` or `oas3`, you simply extend the `oas` ruleset and Spectral will figure out which rules to apply based on given formats.
+From now on, you don't need to worry about `oas2` or `oas3`, you simply extend the `oas` ruleset and Spectral figures out which rules to apply based on given formats.
```

Comment:
Comma splice. Recommendation:

From now on, you don't need to worry about `oas2` or `oas3`. Simply extend the `oas` ruleset and Spectral figures out which rules to apply based on given formats.